spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -6572,14 +6619,14 @@ class at the same level as your application.
 	public class BatchConfiguration { ... }
 ----
 
-NOTE: Depending on the surface area of your application, you may either have a single
+NOTE: Depending on the complexity of your application, you may either have a single
 `ApplicationConfiguration` class for your customizations or one class per domain area
-when it makes sense. The latter approach allows you to enable it in one of your test
+when it makes sense. The latter approach lets you enable it in one of your test
```

Comment:
"test" -> "tests"?

Reply:
Yes. I missed some other things in that sentence, too. Thanks.